### DISCUSSION TITLE
v1.1.0-preview2

### DISCUSSION THREAD

**Discussion by gabbybilka** [microsoft/WindowsAppSDK#2419] [Announcements]
### 1.1 Preview 2 Release of Windows App SDK 

We've just released our second preview of the 1.1 release of the Windows App SDK! This preview includes several new capabilities, including:
- Additional windowing functionality (see [Windows and Messages](https://docs.microsoft.com/windows/win32/api/_winmsg/)) available in AppWindow APIs.
- Tall title bar support for apps drawing custom title bars with AppWindowTitleBar.ExtendsContentIntoTitleBar.
- Improved usability and troubleshooting in Bootstrapper APIs for deployment.
- Several bug fixes across the WinUI and Notifications areas.
- Performance improvements in C# applications.

**Regression from 1.1 Preview 1:** Push notifications are not supported for unpackaged apps. This is expected to be restored in the next release.

To see everything that's new and changed, see the full [release notes here](https://aka.ms/windowsappsdk/1.1/1.1.0-preview2/release-notes). 

### What's next 
To keep up with progress being made on the Windows App SDK, please see our [feature roadmap](https://aka.ms/winappsdkroadmap), which gets updated regularly.  

### Providing feedback
As we introduce new & upcoming features in our preview releases, we appreciate all of your feedback as we work towards a stable release:
- For WinUI 3 feedback, please open an issue on the [WinUI repo](https://github.com/microsoft/microsoft-ui-xaml/issues/new/choose).
- For feedback regarding all other Windows App SDK components, please open an issue here, on the [Windows App SDK repo](https://github.com/microsoft/WindowsAppSDK/issues/new/choose).

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WindowsAppSDK/releases/tag/v1.1.0-preview2'>v1.1.0-preview2</a>.</em>

**Comment by ghost1372** [comment]
Where is mica?

**Reply by gabbybilka** [reply]
Our current plan is to include Mica as part of our next preview release - it just didn't make it in time for this preview. We will let you know if that changes.

**Reply by ghost1372** [reply]
> Our current plan is to include Mica as part of our next preview release - it just didn't make it in time for this preview. We will let you know if that changes.

Tnx but in release note mentioned that

>  This is the latest release of the preview channel for version 1.1

So...

**Reply by gabbybilka** [reply]
Latest, 1.1 preview 2 is the most recent preview release - but it is not our *last* preview release for 1.1😊

**Reply by gabbybilka** [reply]
Does that help clarify?

**Reply by ghost1372** [reply]
> Does that help clarify?

Yes, Thank you🙏

**Reply by ParkhutRoman** [reply]
Finlay fixed ContentWindowSiteBridge ) 
![image](https://user-images.githubusercontent.com/12161245/164353647-4f98addf-66f9-4723-8bda-9b0b7bfda565.png)

**Reply by ParkhutRoman** [reply]
Yep we need Mica from the box ) 
![image](https://user-images.githubusercontent.com/12161245/164354898-933e85f2-88da-4a4b-835a-354f8b307959.png) 
for now, it will somehow work to finally pick up the colors for my app before fixed version

**Comment by Samuel12321** [comment]
When will ShouldConstrainToRootBounds="False" be fixed in WinUI 3?

**Reply by DrusTheAxe** [reply]
That's probably better asked in the [WinUI3 forum](https://github.com/microsoft/microsoft-ui-xaml/issues), if not already discussed. [Searching for ShouldConstrainToRootBounds](https://github.com/microsoft/microsoft-ui-xaml/search?q=ShouldConstrainToRootBounds&type=issues) shows multiple hits so you may find helpful answers there

**Reply by Samuel12321** [reply]
hi @DrusTheAxe thanks for the response, you are correct there are plenty of discussions about this in the winui3 forum but no answers to when this will be fixed.
